specifications/api/swagger.yaml

```diff
@@ -370,127 +370,128 @@ networkInformationSoftwareUpdate: &networkInformationSoftwareUpdate
 
 #############################################################################
 #                                                                           #
-#                                RESOURCES                                  #
+#                              DEFINITIONS                                  #
 #                                                                           #
 #############################################################################
 
-address: &address
```

Comment:
I am assuming you have just relocated and polished a few things here

Reply:
Yes! I actually forgot about this while writing the spec file but, Swagger expects "definitions" of model / resources to be located in a "definitions" properties from the root swagger file. So I just moved things to be located under this field.